library/src/main/java/com/novoda/downloadmanager/PrimaryStorageRoot.java

```diff
@@ -0,0 +1,18 @@
+package com.novoda.downloadmanager;
+
+import android.content.Context;
+
+class PrimaryStorageRoot implements StorageRoot {
+
+    private final Context context;
+
+    PrimaryStorageRoot(Context context) {
+        this.context = context;
+    }
+
+    @Override
+    public String path() {
+        return context.getApplicationContext().getFilesDir().getAbsolutePath();
```

Comment:
we won't use `private static final String DOWNLOADS_DIR = "/downloads/";` anymore?

Reply:
Well I wasn't sure about this tbh. I figured we should just use the internal directory `files` if the client wants to add to the root then they can by supplying their own `StorageRoot`.

Reply:
Yeah I know but this is our implementation, just to align it with what we had before I'd keep the `downloads`. The client can provide whatever then in their own implementation